qlib/mod.rs

```diff
@@ -126,26 +126,47 @@ pub fn InitSingleton() {
     }
 }
 
-pub const HYPERCALL_PANIC: u16 = 2;
```

Comment:
it was tested that 8 byte alignment for MMIO addr is not necessary when MMU is enabled.